doc/modules/covariance.rst

```diff
@@ -190,10 +189,10 @@ The matrix inverse of the covariance matrix, often called the precision
 matrix, is proportional to the partial correlation matrix. It gives the
 partial independence relationship. In other words, if two features are
 independent conditionally on the others, the corresponding coefficient in
-the precision matrix will be zero. This is why it makes sense to estimate
-a sparse precision matrix: by learning independence relations from the
-data, the estimation of the covariance matrix is better conditioned. This
-is known as *covariance selection*.
+the precision matrix will be zero. This is why it makes sense to
+estimate a sparse precision matrix: the estimation of the covariance
+matrix is better conditioned by learning independence relations from
+the data. This is known as *covariance selection*.
```

Comment:
I prefer the previous version here, but it's minor.

Reply:
I'm happy to revert it. I feel that throwing a "learning independence relations" directly breaks the flow of reading. At least that is what happened to me.

Reply:
Well, I find the formulation "This is why it makes sense to estimate [something]: the estimation of [something else] etc"  a bit awkward as well. 

But I'm not a native English speaker - let's wait for a second review...